src/main/java/seedu/cakecollate/ui/OrderCard.java

```diff
@@ -66,6 +68,7 @@ public OrderCard(Order order, int displayedIndex) {
                 .forEach(tag -> tags.getChildren().add(new Label(tag.tagName)));
         deliveryDate.setText(order.getDeliveryDate().toString());
         deliveryStatus.setText(order.getDeliveryStatus().toString());
+        request.setText(order.getRequest().value);
```

Comment:
Maybe you can use toString() here instead of value

Reply:
changed